sambaplayersdk/src/main/res/values/strings.xml

```diff
@@ -31,4 +31,5 @@
     <string name="cancel">Cancelar</string>
     <string name="no_caption">Sem Legendas</string>
     <string name="qualitty_auto">Auto</string>
+    <string name="bitmovin_key">d4b4a5aa-efc6-4672-91de-af9fe1dbdb9f</string>
```

Comment:
@luishenkoti Essa chave da API é um dado sensível e estamos em um repositório público. Nesse caso, por segurança, precisamos de alterar a forma de definição dessa chave.

Uma sugestão seria setar a chave `bitmovin_key` no momento do build utilizando uma variável de ambiente. Não conheço muito de mobile, então não sei se é possível fazer dessa forma, imagino que sim. Se não for possível, é necessário pensar em uma outra forma de proteção desse dado.